projects/ad9656_fmc/zcu102/system_project.tcl

```diff
@@ -0,0 +1,14 @@
+
+source ../../scripts/adi_env.tcl
+source $ad_hdl_dir/projects/scripts/adi_project_xilinx.tcl
+source $ad_hdl_dir/projects/scripts/adi_board.tcl
+
+adi_project ad9656_zcu102
```

Comment:
If the folder name was changed to ad9656_fmc then the project name should also reflect that e.g. ad9656_fmc_zcu102

Reply:
Done it in v7